mediator/med_phases_aofluxes_mod.F90

```diff
@@ -914,6 +1010,20 @@ subroutine med_aofluxes_update(gcomp, aoflux_in, aoflux_out, rc)
        end do
     end if
     if (compute_atm_dens) then
+       if (trim(aoflux_code) == 'ccpp' .and. &
```

Comment:
Can we declare these constants as parameters instead of in-lining them in the code?
0 and 100 and I think 287.058 and 0.608 should already be in a parameter file isn't it?

Reply:
I could try to ingest them to `ufs_const_mod.F90`. Let me look at it.

Reply:
@jedwards4b I checked this again and those are not introduced by me. It is already in CMEPS master. Of curse I could add them to the `ufs_const_mode.F90` but CESM uses `shr_const_mod` which is not part of the CMEPS (as I see). So, let me know your preference and if you want me to add them to the constant file, please point me the correct place of `shr_const_mod`.

Reply:
Maybe just make them parameters in this subroutine?

Reply:
@jedwards4b I follow your suggestions and make most of them parameter. Let me know if it is fine now.